docs/Deploying-Pega-on-PKS.md

```diff
@@ -423,7 +423,7 @@ tier:
     service:
       # Enter the domain name to access web nodes via a load balancer.
       #  e.g. web.mypega.example.com
-      domain: "**pks.web.dev.pega.io**"
+      domain: "**TKGI.web.dev.pega.io**"
```

Comment:
Some of these have switched from lowercase (pks) to uppercase (TKGI).  Is that intentional or a find/replace artifact?

Reply:
this is due to find/replace